community/working-groups.md

```diff
@@ -16,11 +16,12 @@ UTC refers to [Coordinated Universal Time](https://en.wikipedia.org/wiki/Coordin
 | Working Group                    | Slack channel       | Meetings                                 | Contact                                                                  |
 | -------------------------------- | ------------------- | ---------------------------------------- | ------------------------------------------------------------------------ |
 | Astronomy                        | `#astro`            | Fourth Friday of each month at 19:00 UTC | [Mosè Giordano][Mosè-Giordano], [Miles Lucas][Miles-Lucas]               |
```

Comment:
@barrettp took over this one, but I'm not sure what's the new schedule of the meetings

Reply:
The Astronomy meetings are typically the fourth Thursday of each month except when they occur on a holiday such as Thanksgiving in the US or Xmas.

Reply:
@barrettp Can you submit a PR with the update?